app/src/test/java/echo/SocketIOTest.java

```diff
@@ -0,0 +1,50 @@
+package echo;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.Socket;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+
+public class SocketIOTest {
+  SocketIO socketIO = new SocketIO();
+  Socket mockSocket = Mockito.mock(Socket.class);
+  InputStream mockInputStream = mock(InputStream.class);
+
+  @Test
+  @DisplayName("should read a message when a message is sent")
+  public void should_ReadMessage_When_MessageIsSent() throws IOException {
+    String inputString = "Hello\n";
```

Comment:
This is another example of using the type in the variable name. Here, I would suggest to call it "input".